### ISSUE TITLE
5.15.62使用armbian-update更新2022.9.5提示无更新。

### ISSUE THREAD

**Issue by focusos** [ophub/amlogic-s9xxx-armbian#541]
Welcome to Armbian 22.08.0-trunk Bullseye with Linux 5.15.62-flippy-76+o

System load:   37%              Up time:       8 days 7:26
Memory usage:  14% of 1.77G     IP:            172.17.0.1 10.10.10.9 100.83.49.7
CPU temp:      50°C             Usage of /:    43% of 6.4G
RX today:      56.1 GiB

[ General system configuration (beta): armbian-config ]

Last login: Fri Sep  2 22:59:33 2022 from 10.10.10.8
root@Armbian:~# armbian-update
[ STEPS ] Ready to update, please wait...
[ STEPS ] Start initializing the environment...
[ INFO ] Current version: 5.15.62, Latest version: 5.15.62
[ FINISH ] Already the latest version, no need to update.

**Comment by ophub** [comment]
这是最新的了

**Comment by JFLim1** [comment]
Hi @ophub,

Just checking is kernel-5.15.62 going to be upgraded just to 5.15.65 like the other kernel series 5.10.y and 5.19.y?  Thanks.

Are there any issue with kernel 5.15.63, 5.15.64 or 5.15.65 as you seems to have stop at 5.15.62 at this point in time.

**Comment by ophub** [comment]
https://github.com/ophub/kernel/tree/main/pub/dev

`The 5.15 kernel has not displayed HDMI output since 62`, so it is temporarily placed in the dev directory. After the update command, add dev to `armbian-update 5.15.65 dev`

Because the upstream public version of the kernel update does not have HDMI, so users who use desktop systems should not update, the black screen is not displayed. If it is a server version, it can be updated through ssh access. There are currently no problems other than finding no HDMI.

After the upstream is repaired, it will continue to be put back into the `stable` directory

**Comment by JFLim1** [comment]
> After the upstream is repaired, it will continue to be put back into the `stable` directory

Thank you for taking the precaution to cater for Desktop users.  I had encountered the NO HDMI output with kernel-5.15.61-flippy-75+o.  Resolved by after your input that kernel-5.15.62-flippy-76+o.

No wonder Armbian UEFI kernel-5.15.63-arm64 had the NO HDMI output issue and someone also mentioned Armbian UEFI kernel-5.19.5-arm64 also had the NO HDMI output issue.  

I tried Armbian UEFI kernel-5.19.6-arm64 and it works with HDMI output.
